crates/bevy_text/src/lib.rs

```diff
@@ -34,13 +34,18 @@ use bevy_render::{camera::CameraUpdateSystem, ExtractSchedule, RenderApp};
 use bevy_sprite::SpriteSystem;
 use std::num::NonZeroUsize;
 
+/// Adds text rendering support to an app.
+///
+/// When the `bevy_text` feature is enabled with the `bevy` crate, this
+/// plugin is included by default in the `DefaultPlugins`.
 #[derive(Default)]
 pub struct TextPlugin;
 
-/// [`TextPlugin`] settings
+/// Settings used to configure the [`TextPlugin`].
 #[derive(Resource)]
 pub struct TextSettings {
-    /// Maximum number of font atlases supported in a [`FontAtlasSet`].
```

Comment:
These changes are all fine, but `TextSettings` could use some more even.  

Both the original comment and the name 'max_font_atlases` are misleading. The `max_font_atlases` field only sets the limit after which a single warning is emitted but the text pipeline will keep creating new text atlases until it runs out of memory and panics with a wgpu error.